game/src/game/player/player_behaviour.rs

```diff
@@ -52,7 +54,8 @@ impl Plugin for PlayerBehaviorPlugin {
                         .before(player_jump)
                         .run_if(any_matching::<(With<Falling>,)>()),
                 )
-                    .in_set(PlayerStateSet::Behavior),
+                    .in_set(PlayerStateSet::Behavior)
+                    .before(update_sprite_colliders),
```

Comment:
This was ambiguous before, and was actually randomly causing the attack to have no collider, and sometimes a giant box collider, depending on the order execution.